qa/L0_iterative_sequence/iterative_sequence_e2e.py

```diff
@@ -103,30 +103,44 @@ def test_generate_stream(self):
             self.assertEqual(res_count, data["OUTPUT"])
         self.assertEqual(0, res_count)
 
-    def test_grpc_stream(self, sequence_id=0, sequence_start=False):
+    def test_grpc_stream(
+        self, sequence_id=0, sequence_start=False, num_requests=1, validation=True
+    ):
         user_data = UserData()
         with grpcclient.InferenceServerClient("localhost:8001") as triton_client:
             triton_client.start_stream(callback=partial(callback, user_data))
             inputs = []
             inputs.append(grpcclient.InferInput("INPUT", [1, 1], "INT32"))
             inputs[0].set_data_from_numpy(np.array([[2]], dtype=np.int32))
 
-            triton_client.async_stream_infer(
-                model_name="iterative_sequence",
-                inputs=inputs,
-                sequence_id=sequence_id,
-                sequence_start=sequence_start,
-            )
-            res_count = 2
+            for _ in range(num_requests):
+                triton_client.async_stream_infer(
+                    model_name="iterative_sequence",
+                    inputs=inputs,
+                    sequence_id=sequence_id,
+                    sequence_start=sequence_start,
+                )
+            res_count = 2 * num_requests
             while res_count > 0:
                 data_item = user_data._completed_requests.get()
                 res_count -= 1
                 if type(data_item) == InferenceServerException:
                     raise data_item
                 else:
-                    self.assertEqual(res_count, data_item.as_numpy("OUTPUT")[0][0])
+                    if validation:
+                        self.assertEqual(
+                            res_count % 2, data_item.as_numpy("OUTPUT")[0][0]
+                        )
             self.assertEqual(0, res_count)
 
+    def test_backlog_fill(self):
+        config = r'"sequence_batching" : { "iterative_sequence" : true, "max_sequence_idle_microseconds": 8000000, direct: { "max_queue_delay_microseconds" : 10000000 }}'
+        with grpcclient.InferenceServerClient("localhost:8001") as triton_client:
+            triton_client.load_model(
+                "iterative_sequence", config=MODEL_CONFIG_BASE.format(config)
+            )
+        self.test_grpc_stream(num_requests=4, validation=False)
```

Comment:
Why not always "validate" or "not-validate"?

Reply:
This is because the responses get mixed up since we are using a single callback for both requests.